Add May 1 changelog entry (#352)
onkernel/docs: changelog.mdx

For API library updates, see the [Node SDK](https://github.com/onkernel/kernel-node-sdk/blob/main/CHANGELOG.md), [Python SDK](https://github.com/onkernel/kernel-python-sdk/blob/next/CHANGELOG.md), and [Go SDK](https://github.com/onkernel/kernel-go-sdk/blob/main/CHANGELOG.md) changelogs.
</Note>

<Update label="May 1" tags={["Product", "Docs"]}>
## Product updates

> - Added a [`kernel browsers curl`](https://www.kernel.sh/docs/browsers/curl) command to the [CLI](https://github.com/kernel/cli) for making HTTP requests directly from a cloud browser session, so calls go out with the browser's cookies, session state, and network identity instead of your local terminal's.

Reviewer: ```suggestion
- Added [`kernel browsers curl`](https://www.kernel.sh/docs/browsers/curl) command to the [CLI](https://github.com/kernel/cli) for making HTTP requests directly from a cloud browser session using browser's cookies, session state, and network identity instead of your local terminal's.
```

- Extended [Projects](/info/projects) support to the [CLI](https://github.com/kernel/cli) and [MCP server](/reference/mcp-server) — use the `--project` flag to scope CLI commands to a specific project, and MCP tools now resolve projects by name.
- Improved mouse movement timing in cloud browsers with Gaussian-distributed delays between movements for more natural-looking interactions.

> - Profile downloads now extract directly to a local directory, with improved handling of in-progress download responses. The dashboard now respects the active project on download and surfaces errors as toasts.

Reviewer 2: ```suggestion
```

- Improved managed auth UX: MFA prompts now display which channel (email or phone) the verification code was sent to, and `--credential-auto` now defaults to enabled when `--credential-provider` is set.
- Added a dedicated `switch` MFA option type so generic method-switcher links ("Use another method", "Try another way") are distinguished from buttons that name a specific method, improving auto-click reliability on multi-step MFA pages.
- Managed auth CUA flow now resolves external credentials (e.g. 1Password) via auto-lookup.
- Managed auth connections now expose `browser_session_id` so callers can inspect or terminate the underlying browser session via the `/browsers` API.

> - Added a `diff-profile-archives` skill to the [Kernel skills repo](https://github.com/kernel/skills) for comparing Chrome profile archives directly from Claude Code.

Reviewer 2: ```suggestion
- Added a `diff-profile-archives` skill to the [Kernel skills repo](https://github.com/kernel/skills) for comparing Chrome profile archives directly with a coding agent.
```
